tools/slang-test/main.cpp

```diff
@@ -74,9 +198,284 @@ struct Options
     Dictionary<TestCategory*, TestCategory*> excludeCategories;
 
     // By default we can test against all apis
-    int enabledApis = int(RenderApiFlag::AllOf);           
+    int enabledApis = int(RenderApiFlag::AllOf);
 };
-Options options;
+
+// Globals
+
+Options g_options;
```

Comment:
The convention currently in the code leans more toward `gOptions` (and `mSomeMember` for that matter). Not worth delaying checkin, but we'll need to converge on this stuff sooner or later.